tools/workspace-plugin/src/generators/split-library-in-two/generator.ts

```diff
@@ -1,17 +1,560 @@
-import { addProjectConfiguration, formatFiles, generateFiles, Tree } from '@nx/devkit';
-import * as path from 'path';
+import {
```

Comment:
Let's add a README for this generator like we do for the others

Reply:
we can, I didn't add it because this shouldn't be invoked by standard users, it will be removed once we migrate all packages.